### ISSUE TITLE
Compatibility with Jekyll 4.0?

### ISSUE THREAD

**Issue by etagwerker** [poole/hyde#253]
Hi there,

I was trying to install `hyde` but I ran into this issue:

```
$ bundle
Fetching gem metadata from https://rubygems.org/..........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "jekyll":
  In snapshot (Gemfile.lock):
    jekyll (= 4.0.0)

  In Gemfile:
    blog was resolved to 1.0.0, which depends on
      jekyll (~> 4.0.0)

    jekyll-theme-hyde was resolved to 2.0.0, which depends on
      jekyll (~> 3.3)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

It seems that `hyde` is not compatible with Jekyll 4.0? Would you be interested in a PR to make it compatible? 

Please let me know.

Thanks!

**Comment by mdo** [comment]
Should be fixed with #258

**Comment by mralusw** [comment]
It doesn't seem fixed at all.  `bundle add jekyll-theme-hyde` gives

```
Resolving dependencies...
Bundler could not find compatible versions for  gem "jekyll":
  In Gemfile:
    jekyll (~> 4.1.0)

jekyll-theme-hyde was resolved to 2.0.0,
which depends on
      jekyll (~> 3.3)
```
